src/assets/scripts/client/game/GameOptions.js

```diff
@@ -63,15 +67,21 @@ export default class GameOptions {
     }
 
     /**
+     * Sets a game option to a given value
+     *
      * @for GameOptions
      * @method set
-     * @param name {string}
-     * @param value
+     * @param name {string} name of the option to change
+     * @param value {string} value to set the option to
      */
     set(name, value) {
         localStorage[`zlsa.atc.option.${name}`] = value;
         this[name] = value;
 
+        if (this._options[name].onChangeEvent) {
```

Comment:
ref `_noop()` comment from way up.

I would actually advocate for not including the `onChangeEvent` key at all if it isn't needed.  Simply check for the key and then create the handler only if the key is present.

I would also suggest a name change to `onChangeEventHandler`